constant.py

```diff
@@ -0,0 +1,36 @@
+"""Quick-Cart constant File to Store Constant Variable & Messages"""
+
+import os
+
+BASE_JSON_DIR = os.path.join(os.getcwd(), "quick_tickets_json")
+TRANSACTION_FILENAME_FORMAT = "{username}_transactions.json"
+DATA_JSON_PATH = os.path.join(BASE_JSON_DIR, "data.json")
+DATA_JSON_FORMAT = {"USER_JSON": {}, "MOVIE_JSON": {}}
+DATA_OPTION = dict(enumerate(["USER_JSON", "MOVIE_JSON"], 1))
+USER_JSON = DATA_OPTION.get(1)
+MOVIE_JSON = DATA_OPTION.get(2)
+
+
+# User Role
+ROLE = tuple(enumerate(["host", "watcher"], 1))
+ROLE_HOST = ROLE[0][1]
+ROLE_WATCHER = ROLE[1][1]
+
```

Comment:
why take role like this ?
you can take dict as well.

Reply:
Updated